v3/spring-cloud-aws-v3-autoconfigure/src/test/java/io/awspring/cloud/v3/autoconfigure/RegionProviderAutoConfigurationTests.java

```diff
@@ -51,7 +54,7 @@ void autoDetectionConfigured_emptyStaticRegionConfigured_DefaultAwsRegionProvide
 	@Test
 	void staticRegionConfigured_staticRegionProviderWithConfiguredRegionConfigured() {
 		this.contextRunner.withPropertyValues("spring.cloud.aws.region.static:eu-west-1").run((context) -> {
-			StaticRegionProvider regionProvider = context.getBean(StaticRegionProvider.class);
+			AwsRegionProvider regionProvider = context.getBean(AwsRegionProvider.class);
```

Comment:
We should also check here that it is an instance of static region provider.

Reply:
Thank you @maciejwalkowiak, solved ;)